batchservice/src/main/java/nu/yona/server/batch/quartz/TriggerController.java

```diff
@@ -0,0 +1,147 @@
+package nu.yona.server.batch.quartz;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.hateoas.ExposesResourceFor;
+import org.springframework.hateoas.Link;
+import org.springframework.hateoas.Resource;
+import org.springframework.hateoas.Resources;
+import org.springframework.hateoas.mvc.ControllerLinkBuilder;
+import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import nu.yona.server.rest.JsonRootRelProvider;
+
+@Controller
+@ExposesResourceFor(CronTriggerDto.class)
+@RequestMapping(value = "/scheduler/triggers", produces = { MediaType.APPLICATION_JSON_VALUE })
+public class TriggerController
+{
+	@Autowired
+	private TriggerManagementService triggerManagementService;
+
+	@RequestMapping(value = "/cron/", method = RequestMethod.GET)
+	@ResponseBody
+	public HttpEntity<Resources<TriggerResource>> getAllTriggers()
+	{
+		return new ResponseEntity<>(createTriggersCollectionResource(triggerManagementService.getAllTriggers(),
+				getAllTriggersLinkBuilder().withSelfRel()), HttpStatus.OK);
+	}
+
+	@RequestMapping(value = "/cron/{group}/", method = RequestMethod.GET)
+	@ResponseBody
+	public HttpEntity<Resources<TriggerResource>> getTriggersInGroup(@PathVariable String group)
+	{
+		return new ResponseEntity<>(createTriggersCollectionResource(triggerManagementService.getTriggersInGroup(group),
+				getTriggersInGroupLinkBuilder(group).withSelfRel()), HttpStatus.OK);
+	}
+
+	@RequestMapping(value = "/cron/{group}/", method = RequestMethod.PUT)
```

Comment:
Same here, this is the only method used? And only with `group`=`OTHER`? Why not simplify this and remove the group layer?